CMake/Dependencies/libwebsockets-CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ ExternalProject_Add(project_libwebsockets
         -DCMAKE_INSTALL_PREFIX=${OPEN_SRC_INSTALL_PREFIX}
         -DCMAKE_C_FLAGS=${CMAKE_C_FLAGS}
         -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+        -DDISABLE_WERROR=ON
```

Comment:
Could you share which error we're seeing here?

Can't we also make this one ON only the failing builds (mac), conditional?

Reply:
Added in description, but it's not on mac, this fails on mips gcc older toolchains and I think it's specific with uclibc toolchains (this was required to build in Ingenic T31) with gcc 5.4 uclibc toolchain. Details and link to lws issue has been added to the description.